src/overlays/actors/ovl_Bg_Bom_Guard/z_bg_bom_guard.c

```diff
@@ -1,13 +1,11 @@
 /*
  * File: z_bg_bom_guard.c
  * Overlay: Bg_Bom_Guard
- * Description: Bombchu Bowling Alley Game
+ * Description: Bombchu Bowling Alley Walls
  */
 
 #include "z_bg_bom_guard.h"
-
 #include "overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.h"
-
 #include <vt.h>
```

Comment:
It doesn't really matter, but fyi I separated these includes since they don't really belong in the same category imo. In particular we generally have the include for the header corresponding to the current file first on its own, then the rest of the includes (in terms of C conventions). But it's not a huge deal either, and we can figure out how we want to handle this once we start using separate headers for everything.

Reply:
ok, yeah i prefer all the includes in one section, but we can definitely coordinate that better down the line